scripts/add_translation.py

```diff
@@ -71,18 +80,47 @@ def get_credentials():
     return credentials
 
 
+def display_languages(service):
+    print 'List of available languages:\n----------------------------'
+    data = service.files().list(q="'" + TRANSLATION_FOLDER_ID + "'" + " in parents").execute()
+    for f in data['files']:
+        if f['mimeType'].endswith('.folder'):
+            LANGUAGE_LIST[f['name']] = f['id']
+
+    # sort the languages alphabetically
+    sorted_list = [e for e in sorted(LANGUAGE_LIST)]
+    id = 1
```

Comment:
General python convention: there is a builtin function called `id`; we generally avoid naming variables the same thing as builtin functions.

(python will allow it, but sometimes this can lead to subtle and hard to trace bugs)